### PR TITLE
Add all available retrievable properties in search for chatMessage

### PR DESCRIPTION
**Pull request by yiwenwang-ms**
Add all available retrievable properties in search for chatMessage

### CODE REVIEW

File: concepts/search-concept-chat-messages.md [hunk 1]
```diff
@@ -381,6 +381,24 @@ Content-type: application/json
 - Sorting results is not supported for messages.
 - You can't use this API with other entity types at this time.
 
+## JSON representation
+
+The following is a JSON representation of all retrievable properties in search to [chatMessage](/graph/api/resources/chatmessage)
```
Comment (njerigrevious) [inline concepts/search-concept-chat-messages.md:386]: Please update to "The following is a JSON representation of all retrievable properties currently available for [chatMessage] (/graph/api/resources/chatmessage) search"

**Review by njerigrevious — CHANGES_REQUESTED** [review]
Let's update the language before finalizing. Thanks!

**Review by njerigrevious — CHANGES_REQUESTED** [review]
Add last minor punctuation change and then you can merge. Thanks!

**Review by dodaromike — APPROVED** [review]

File: concepts/search-concept-chat-messages.md [hunk 2]
```diff
@@ -381,6 +381,24 @@ Content-type: application/json
 - Sorting results is not supported for messages.
 - You can't use this API with other entity types at this time.
 
+## JSON representation
+
+The following is a JSON representation of all retrievable properties currently available for [chatMessage] (/graph/api/resources/chatmessage) search
```
[the remark this answers is not in the document]
Comment (yiwenwang-ms) [reply concepts/search-concept-chat-messages.md]: Done.

**Pull request merged**